src/components/Movies.jsx

```diff
@@ -17,9 +19,15 @@ const Movies = () => {
   }, [])
 
   return (
-    <div>
+    <div className="movie">
       {trendingMovies.map((movie) => (
-        <span key={movie.id}>{movie.original_title}</span>
+        <Card
+          key={movie.id}
+          id={movie.id}
+          poster={movie.poster_path}
+          original_title={movie.original_title}
+          release_date={movie.release_date}
```

Comment:
```suggestion
          title={movie.original_title}
          releaseDate={movie.release_date}
```

As props não precisam ter o mesmo estilo/nome da resposta da API, mas sim algo que faça sentido pro estilo usado no projeto